### DISCUSSION TITLE
🎉 New Release: Dataview, JavaScript, and Templater support (v1.15)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#171] [Announcements]
See the [full release notes](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.15.0).

**Feedback wanted 💬**

I would really appreciate your questions or feedback about scripting in [this discussion thread](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/171), or using [this Google feedback form ↗](https://docs.google.com/forms/d/e/1FAIpQLSeVWHVnookJr8HVQywk5TwupU-p7vkRkSt83Q5jscR6VwpZEQ/viewform?usp=sf_link).

1. Do your existing scripts work as expected? (If not, what changes did you have to make?)
2. Does configuration make sense? (Should I add or change anything?)
3. Try out the Note Toolbar Script components for output, and let me know what you think.
4. Does the documentation and help text make sense? (Should I add or change anything?)

_Thank you!_

**Comment by chrisgurney** [comment]
Example scripts have been added here:
https://github.com/chrisgurney/obsidian-note-toolbar/tree/master/examples/Scripts

**Comment by chrisgurney** [comment]
@Moyf Thanks for trying out the scripting functionality! 

How did it go for you? Did it work as expected? Did you have to make any changes to your scripts?

**Reply by Moyf** [reply]
> @Moyf Thanks for trying out the scripting functionality!
> 
> How did it go for you? Did it work as expected? Did you have to make any changes to your scripts?

Everything works fine for now! Really easy and smooth 😄
( I can finally get rid of other plugins to run TP script in NTB haha)

**Comment by chrisgurney** [comment]
The v1.15 release is now available in Community Plugins. 

Thank you for your patience!

**Reply by Moyf** [reply]
Nice!!
